L1Trigger/TrackFindingTracklet/test/HybridTracksNewKF_cfg.py

```diff
@@ -0,0 +1,98 @@
+  #'/store/relval/CMSSW_11_3_0_pre6/RelValDisplacedMuPt2To100Dxy100/GEN-SIM-DIGI-RAW/113X_mcRun4_realistic_v6_2026D76noPU-v1/00000/011da61a-9524-4a96-b91f-03e8690af3bd.root'
+  '/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/00026541-6200-4eed-b6f8-d3a1fd720e9c.root',
+  '/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/013d0125-8f6e-496b-8335-614398c9210d.root',
+  '/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/058bd134-86de-47e1-bcde-379ed9b79e1b.root',
+  '/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/0915d66c-cbd4-4ef6-9971-7dd59e198b56.root',
+  '/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/09823c8d-e443-4066-8347-8c704929cb2b.root',
+  '/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/0c39a1aa-93ee-41c1-8543-6d90c09114a7.root',
+  '/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/0fcdcc53-fb9f-4f0b-8529-a4d60d914c14.root',
+  '/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/16760a5c-9cd2-41c3-82e5-399bb962d537.root',
+  '/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/1752640f-2001-4d14-9276-063ec07cea92.root',
+  '/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/180712c9-31a5-4f2a-bf92-a7fbee4dabad.root'
+]
+options.register( 'inputMC', inputMC, VarParsing.VarParsing.multiplicity.singleton, VarParsing.VarParsing.varType.string, "Files to be processed" )
+# specify number of events to process.
+options.register( 'Events',100,VarParsing.VarParsing.multiplicity.singleton, VarParsing.VarParsing.varType.int, "Number of Events to analyze" )
+options.parseArguments()
+
+process.options = cms.untracked.PSet( wantSummary = cms.untracked.bool(False) )
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(options.Events) )
+process.source = cms.Source(
+  "PoolSource",
+  fileNames = cms.untracked.vstring( options.inputMC ),
+  #skipEvents = cms.untracked.uint32( 1 ),
+  secondaryFileNames = cms.untracked.vstring(),
+  duplicateCheckMode = cms.untracked.string( 'noDuplicateCheck' )
+)
+process.Timing = cms.Service( "Timing", summaryOnly = cms.untracked.bool( True ) )
+process.MessageLogger.cerr.enableStatistics = False
+process.TFileService = cms.Service( "TFileService", fileName = cms.string( "Hist.root" ) )
+
+if (False):
+  process.writeDataset = cms.OutputModule("PoolOutputModule",
```

Comment:
This fails if False changed to True. To fix, add missing line:
  process.load('Configuration.EventContent.EventContent_cff')
In addition, this writes a huge output dataset. so I suggest, copying value of outputCommands from HybridTracks_cfg.py (though keeping the TTStubs too).

Reply:
You mean those https://github.com/cms-L1TK/cmssw/blob/tschuh12/L1Trigger/TrackFindingTracklet/test/HybridTracksNewKF_cfg.py#L95-L96 which are already there?

Reply:
The line "outputCommands = process.RAWSIMEventContent.outputCommands," adds huge numbers of collections to the output dataset, most of which are not interesting for L1 tracking studies. HybridTracks_cfg.py does not do this, which I think may be better.

Reply:
Since I keep only TTSubs and TTTracks I don't understand what you mean with huge numbers of collections.

Reply:
The line "outputCommands = process.RAWSIMEventContent.outputCommands" adds hundreds of collections to the output, as you can check by changing "False" to "True", and then typing "python -i HybridTracksNewKF_cfg.py" followed by "process.writeDataset", which prints the contents of this python variable.

Reply:
done